### DISCUSSION TITLE
Going on Vacation for Two Months | /en/2023/06/on-vacation/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1492] [Comments]
Believe it or not, the best break I had over the last few years was when I got COVID in December last year. Physically I felt quite painful, but mentally I was completely relaxed in those three days. &hellip;

https://yihui.org/en/2023/06/on-vacation/

**Comment by daijiang** [comment]
Have a safe trip home! I have not been back for 8 years now...

**Reply by yihui** [reply]
Thank you! Hope you can go back soon!

**Comment by shrektan** [comment]
Haha, I'm looking forward to your updates on the Chinese blog. I always enjoy reading the new articles you post there. By the way, if you ever plan to visit Beijing, please don't hesitate to let me know!

**Reply by yihui** [reply]
I don't plan to visit Beijing. If you are able to come to Qingdao in June or Yichang in July, I'll be happy to meet you. I might also spend a week or ten days in Sichuan or Chongqing in mid-July. If you go home at that time, I can also meet you there.

**Reply by liechi** [reply]
According to the weather of the last two years, it would be burning hot in Sichuan and Chongqing in mid-July. Take care.

**Comment by JinjiPang** [comment]
I've been dreaming about how things will be like when I go back to China.

**Comment by hyan87** [comment]
Hi, Yihui. Nice to hear that you have come back to YC city. 
I wrote to you via email, even before I had gotten this news. 
A surprising coincidence.
Have a nice trip~

**Reply by yihui** [reply]
Thanks! I'll leave Yichang on this Saturday.

**Reply by yzng2s** [reply]
Hi, Yihui. I am in Chongqing now, have a nice day.

**Reply by yihui** [reply]
Unfortunately, I didn't go to Chongqing. I'll fly to the US in a few days.
